### ISSUE TITLE
Logic to find the difference in Example 20 is breaking

### ISSUE THREAD

**Issue by viveksacademia4git** [josdejong/jsoneditor#1193]
Logic to find the difference in [Example 20](https://github.com/josdejong/jsoneditor/blob/master/examples/20_custom_css_style_for_nodes.html) is breaking.

I replaced the values of `jsonLeft` [L123](https://github.com/josdejong/jsoneditor/blob/master/examples/20_custom_css_style_for_nodes.html#L123) and `jsonRight` [L140](https://github.com/josdejong/jsoneditor/blob/master/examples/20_custom_css_style_for_nodes.html#L140) with JSON value given below.  
Even though both the objects have the same JSON value, the example implementation indicates that there is a difference, as shown in the screenshot.

```
 jsonLeft =   {
      items: [
        {
          key1: '0.5',
          key2: 'hello'
        },
        {
          key1: '0.5',
          key2: 'world'
        }
      ]
    };

 jsonRight =   {
      items: [
        {
          key1: '0.5',
          key2: 'hello'
        },
        {
          key1: '0.5',
          key2: 'world'
        }
      ]
    };
```

![image](https://user-images.githubusercontent.com/45398326/102702888-13bb3500-4268-11eb-93bf-8d7763c585fd.png)

**Comment by josdejong** [comment]
Thanks for reporting. I've simplified and fixed the diffing code in the example (see 77d96242c06906b0ca9dc58ed466bec70c2ec912):

```js
  function onClassName({ path, field, value }) {
    const leftValue = _.get(jsonRight, path)
    const rightValue = _.get(jsonLeft, path)

    return _.isEqual(leftValue, rightValue)
      ? 'the_same_element'
      : 'different_element'
  }
```

Please note that the example is not a full-fledged, performant diffing algorithm. It's just a small example to show how `onClassName` works.

**Comment by viveksacademia4git** [comment]
This is really good!
